src/FsAutoComplete/CommandInput.fs

```diff
@@ -230,9 +238,9 @@ module CommandInput =
     match inputString with
     | null -> Quit
     | input ->
-      let reader = Parser.createForwardStringReader input 0
-      let cmds = compilerlocation <|> helptext <|> declarations <|> lint <|> registerAnalyzer <|> unusedDeclarations <|> simplifiedNames <|> unusedOpens <|> parse <|> project <|> completionTipOrDecl <|> quit <|> colorizations <|> workspacePeek <|> workspaceLoad <|> fsdn <|> error
-      let cmd = reader |> Parser.getFirst cmds
+      let reader = Parsing.createForwardStringReader input 0
```

Comment:
i expect here to see as change only `dotnetnewlist <|> ` the others are merge error